tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -15,28 +15,38 @@ public class ApiApproval
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void ApproveApi()
+        public void ApproveApi45()
         {
-            Approvals.Verify(PublicApiGenerator.GetPublicApi(typeof(A).Assembly));
+            ApproveApi("net45");

```

Comment:
I know, it's a little less efficient than just using the assembly that's in memory. I can revert, if you like.